assembly/std/index.ts

```diff
@@ -426,3 +426,14 @@ export function fromBytes(arr: StaticArray<u8>): string {
   memory.copy(changetype<usize>(str), changetype<usize>(arr), arr.length);
   return str;
 }
+
+/**
+ * Constructs an event given a key and arguments
+ *
+ * @param {string} key - event key
+ * @param {Array} args - array of string arguments.
+ * @return {string} stringified event.
```

Comment:
I am anticipating new typedoc rules

```suggestion
 * @param key - event key
 * @param args - array of string arguments.
 * @returns stringified event.
```